policies/yaml_schema.rego

```diff
@@ -1,15 +1,52 @@
-package before_resolution
+package yaml_schema
```

Comment:
This will test things, but Weaver will no longer work with these package renames.

Reply:
Indeed. See my other comment on the same line.

Reply:
great catch, thank you both